src/types/ApplicationForm.ts

```diff
@@ -17,6 +17,18 @@ export interface ApplicationForm {
   readonly createdAt: Date;
```

Comment:
One more thing might be to make fields mandatory here in the original type, then it really cannot be undefined under any circumstances.

Reply:
Doing that causes validation failures that make me think an empty array does not pass validation unless there is something else I'm missing.